eternaleconomy-core/src/main/java/com/eternalcode/economy/format/DecimalUnit.java

```diff
@@ -0,0 +1,24 @@
+package com.eternalcode.economy.format;
+
+import java.io.Serializable;
+
+@SuppressWarnings("ClassCanBeRecord")
+public class DecimalUnit implements Serializable {
```

Comment:
Serializable?

Reply:
When you use Serializable or OkaeriConfigs here, you don't need to serialize it manually.